examples/nequip/README.md

```diff
@@ -0,0 +1,47 @@
+# NequIP models in OpenMM-ML
+
+This directory contains examples for running simulations using a potential generated through the NequIP framework.
+
+## Installation
+
+First install the `openmm-torch` and `nnpops` packages from conda-forge:
+
+```
+conda install -c conda-forge openmm-torch nnpops
+```
+
+Then install the development versions of NequIP and `openmm-ml` using pip:
```

Comment:
Makes sense. I rewrote the instructions so that that the packages from the OpenMM ecosystem, viz. `NNPOps` and `openmm-ml` packages, are installed from conda-forge.

Regarding the pre-release of NequIP, the one currently available through pip is version 0.5.6, while `pip install git+https://github.com/mir-group/nequip@develop` installs version 0.6.0. There's some discussion in [this thread](https://github.com/mir-group/nequip/issues/288) as to why the development version of NequIP might be better (or necessary) to use with this interface. I am sure @Linux-cpp-lisp is in a better position to provide more informed answers to your questions. @Linux-cpp-lisp could you please clarify why is the development version of NequiP required and whether there any plans to make it available through pip and/or conda? Many thanks!

Reply:
Sorry for coming around to this thread late, and thanks both for your efforts on this!

This is something I need to fix and hope to have fixed and available normally on PyPI in the very near future;  I'll let you know as soon as I have that up.  Since you are using `load_deployed_model`, this will probably work with the current `main` as well, but ideally I will just get that released and we will restrict to `>=0.6.0`.  Thanks for your flexibility while I clean this up.

Reply:
OK, 0.6.0 is now available from PyPI: https://pypi.org/project/nequip/

Let me know if this resolves the issues for you.

Reply:
(I would recommend restricting the `nequip` version to 0.6.0+ just for simplicity's sake)

Reply:
Many thanks @Linux-cpp-lisp. That's really useful! I'll integrate the new NequIP version soon.

Reply:
Great!